tests/integration/golang/mlflow/run/log_batch_test.go

```diff
@@ -215,6 +215,35 @@ func (s *LogBatchTestSuite) TestMetrics_Ok() {
 				"key2": 4,
 			},
 		},
+		{
+			name: "LogMany",
+			request: &request.LogBatchRequest{
+				RunID: s.run.ID,
+				Metrics: func() []request.MetricPartialRequest {
+					metrics := make([]request.MetricPartialRequest, 100*1000)
+					for k := 0; k < 100; k++ {
```

Comment:
why it couldn't be just one cycle?

Reply:
It seems like LogMany would want more than one, so 2x?

Reply:
I'm not entirely sure I understand your question. Is it about merging LogSeveral and LogMany? Or is it about doing 100 metrics with 1000 values each?

Reply:
my question was really just responding to Dmitriy's question :-)

Reply:
Oh I was asking @dsuhinin 😅

Reply:
no no, never mind. It seems like a got the idea. sorry for the late answer.